src/runtime_src/core/pcie/driver/linux/xocl/subdev/ddr_srsr.c

```diff
@@ -25,9 +25,8 @@
 #define	REG_CALIB_OFFSET		0x00000008
 #define	REG_XSDB_RAM_BASE		0x00004000
 
-#define	XSDB_RAM_MAX_IN_WORDS		2901
+#define	XSDB_RAM_MAX_IN_WORDS		2902
```

Comment:
we still can't get this number from HW?

Reply:
done, get the size from XSDB offset +4 & +8